test/exercises/duplicates/forStudents/duplicates.chpl

```diff
@@ -0,0 +1,117 @@
+// This version is perhaps the most obvious / easiest to parallelize
+
```

Comment:
We may want to follow the style from other exercises, and add a header comment here:

```chpl
// STUDENTS: Browse through the code below to your heart's content.
// When you're ready to start coding, jump down to main() and read
// through the comments labeled STEP 0, STEP 1, STEP 2, etc.  These
// will lead you through a number of coding steps and transformations.
```